### ISSUE TITLE
repo.saltstack.com should have python-urllib3 and python-backports_ssl_match_hostname

### ISSUE THREAD

**Issue by ahammond** [saltstack/salt#27427]
```
[root@hss62 ~]# yum install salt
Loaded plugins: fastestmirror, presto
Loading mirror speeds from cached hostfile
Setting up Install Process
Resolving Dependencies
--> Running transaction check
---> Package salt.x86_64 0:2015.5.5-1 will be updated
---> Package salt.noarch 0:2015.8.0-2.el6 will be an update
--> Processing Dependency: python-tornado >= 4.2.1 for package: salt-2015.8.0-2.el6.noarch
--> Processing Dependency: python-requests >= 1.0.0 for package: salt-2015.8.0-2.el6.noarch
--> Processing Dependency: python-futures >= 2.0 for package: salt-2015.8.0-2.el6.noarch
--> Processing Dependency: python-crypto >= 2.6.1 for package: salt-2015.8.0-2.el6.noarch
--> Processing Dependency: python-jinja2 for package: salt-2015.8.0-2.el6.noarch
--> Processing Dependency: pciutils for package: salt-2015.8.0-2.el6.noarch
--> Processing Dependency: PyYAML for package: salt-2015.8.0-2.el6.noarch
--> Running transaction check
---> Package PyYAML.x86_64 0:3.11-1.el6 will be installed
--> Processing Dependency: libyaml-0.so.2()(64bit) for package: PyYAML-3.11-1.el6.x86_64
---> Package pciutils.x86_64 0:3.1.4-11.el6 will be installed
---> Package python-crypto.x86_64 0:2.6.1-2.el6 will be installed
---> Package python-futures.noarch 0:3.0.3-1.el6 will be installed
---> Package python-jinja2.x86_64 0:2.2.1-2.el6 will be installed
--> Processing Dependency: python-babel >= 0.8 for package: python-jinja2-2.2.1-2.el6.x86_64
---> Package python-requests.noarch 0:0.11.1-2.el6 will be updated
---> Package python-requests.noarch 0:1.1.0-3.el6 will be an update
--> Processing Dependency: python-urllib3 for package: python-requests-1.1.0-3.el6.noarch
--> Processing Dependency: python-ordereddict for package: python-requests-1.1.0-3.el6.noarch
--> Processing Dependency: python-chardet for package: python-requests-1.1.0-3.el6.noarch
---> Package python-tornado.x86_64 0:4.2.1-1.el6 will be installed
--> Processing Dependency: python-backports-ssl_match_hostname for package: python-tornado-4.2.1-1.el6.x86_64
--> Running transaction check
---> Package libyaml.x86_64 0:0.1.3-1.el6 will be installed
---> Package python-babel.noarch 0:0.9.4-5.1.el6 will be installed
---> Package python-chardet.noarch 0:2.0.1-1.el6 will be installed
---> Package python-ordereddict.noarch 0:1.1-2.el6 will be installed
---> Package python-requests.noarch 0:1.1.0-3.el6 will be an update
--> Processing Dependency: python-urllib3 for package: python-requests-1.1.0-3.el6.noarch
---> Package python-tornado.x86_64 0:4.2.1-1.el6 will be installed
--> Processing Dependency: python-backports-ssl_match_hostname for package: python-tornado-4.2.1-1.el6.x86_64
--> Finished Dependency Resolution
Error: Package: python-requests-1.1.0-3.el6.noarch (SaltStack)
           Requires: python-urllib3
Error: Package: python-tornado-4.2.1-1.el6.x86_64 (SaltStack)
           Requires: python-backports-ssl_match_hostname
 You could try using --skip-broken to work around the problem
[root@hss62 ~]# 
```

**Comment by jfindlay** [comment]
@ahammond, thanks for the report.  What system and system version are you installing salt on.  Do you have EPEL installed?

**Comment by ahammond** [comment]
Centos 6.2 and epel is on there. And yeah, Centos 6.2 is ancient. Believe
me I know.

**Comment by dmurphy18** [comment]
@ahammond python-requests and python-urllib3 are available from the Centos 6 base (yum list on 6.7, version readily available), python-backports-ssl_match_hostname is available from Centos 6 extras. How are newer packages made available to your systems, given your system is Centos 6.2 and a yum update will move your system to 6.7.

**Comment by ahammond** [comment]
We don't have the updates repo enabled because of hysterical... er... historical reasons (some of our code is fragile and we can't afford to fix it right now). Guess I'm stuck rolling my own packaging for these ones.

**Comment by dmurphy18** [comment]
@ahammond We are looking at supplying some of these dependencies which are available from the OS base repo, hence the interest in how your systems are updated.  The worry here is that there may be some incompatibility with later versions of these dependencies (e.g. libc, python should be fine) on older versions of the OS.

**Comment by ahammond** [comment]
It'd be really nice to have those python deps in place. We're working on
moving forward to CentOS 6.7 and eventually 7, but that's not gonna be
tomorrow. :)
